extensions/interactions/ImageClickInput/ImageClickInput.py

```diff
@@ -59,6 +59,7 @@ class ImageClickInput(base.BaseInteraction):
         'options': {
             'x_axis_label': 'Answer',
             'y_axis_label': 'Count',
+            'show_addressed_column': False,
```

Comment:
There isn't really an "addressed column" in a bar chart...

Reply:
(As in, does this option even need to be specified?)

Reply:
Done, renamed to a better name (that may be useful later).